protoc_plugin/lib/src/file_generator.dart

```diff
@@ -644,7 +644,6 @@ class FileGenerator extends ProtobufContainer {
 //  Generated code. Do not modify.
 //  source: ${descriptor.name}
 //
-// @dart = 2.12
```

Comment:
I've removed all of them, since they would mess up the enum files and I don't think they serve any purpose.